javascript/hangman.js

```diff
@@ -2,54 +2,119 @@ class Hangman {
   constructor(words) {
     this.words = words;
     // ... your code goes here
+    this.secretWord = this.pickWord();
+    this.letters = [];
+    this.guessedLetters = "";
+    this.errorsLeft = 10;
   }
 
   pickWord() {
     // ... your code goes here
+    return this.words[Math.floor(Math.random() * this.words.length)];
   }
 
   checkIfLetter(keyCode) {
     // ... your code goes here
+    if (keyCode >= 65 && keyCode <= 90) return true;
+    else return false;
   }
 
   checkClickedLetters(letter) {
     // ... your code goes here
+    if (this.letters.includes(letter) || this.guessedLetters.includes(letter)) return false;
```

Comment:
Podemos hacer la verificación mas corta:
```suggestion
    return !this.letters.includes(letter);
```